homeassistant/components/lupusec/strings.json

```diff
@@ -0,0 +1,40 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Lupus Electronics LUPUSEC connection",
+        "data": {
+          "host": "Hostname/IP",
+          "username": "Username",
+          "password": "Password",
+          "name": "Name"
+        }
+      }
+    },
+    "error": {
+      "connection": "Failed to connect, check hostname, username and password."
+    },
+    "abort": {
+      "single_instance_allowed": "Only a single instance is allowed!"
```

Comment:
Currently you are not using this. I think we can add support for more instances with ease if there is no technical reason it's blocked

Reply:
Removed and support multiple instances.